lib/iris/tests/unit/fileformats/grib/grib_phenom_translation/test_grib_phenom_translation.py

```diff
@@ -0,0 +1,168 @@
+# (C) British Crown Copyright 2014 - 2017, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+'''
+Unit tests for the mod:`iris.fileformats.grib.grib_phenom_translation` module.
+
+Carried over from old iris/tests/test_grib_phenom_translation.py.
+Code is out of step with current test conventions and standards.
```

Comment:
This does look almost identical to iris/tests/test_grib_phenom_translation.py, so either this or that file should be removed (I vote that file).